packages/service/tests/acceptance/test_session_management.py

```diff
@@ -20,6 +20,7 @@
 from tests.utilities.stubs.pinaware.types_pb2 import Configurations, Outputs
 
 
+@pytest.mark.usefixtures("filter_wrong_configurations_message_type_warnings")
```

Comment:
FYI, you can single-source this for the file with `pytestmark = pytest.mark.usefixtures("filter_wrong_configurations_message_type_warnings")`

Reply:
The docs for pytest.mark.filterwarnings also point this out.